pages/cart/cart.css

```diff
@@ -0,0 +1,75 @@
+.cart-page{
+    background: var(--lightergrey);
+    padding: 8rem 0px;
+    height: 100vh;
+}
+.cart-content{
+    display: flex;
+    gap: 20px;
+    justify-content: center;
+}
+.flex-between{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 1rem 0px;
+}
+.horizontal-card{
+    display: flex;
+    height: max-content;
+}
+.card-details{
+    padding: 0rem 2rem;
+    background: var(--white);
+}
+.price-details{
+    width: clamp(200px, 40vw, 300px);
+    padding: 2rem;
+    background: var(--white);
+}
+.text-right{
+    text-align: right;
+}
+.oldprice{
+    color: var(--grey);
+    text-decoration: line-through;
+}
+.quantity{
+    display: flex;
+    gap: 20px;
+    align-items: center;
+}
+.qty-ip{
+    width: 2rem;
+    border: none;
+    outline: none;
+    text-align: center;
+    padding: 4px 8px;
```

Comment:
Try to follow one pattern, either rem or px.

Reply:
Done [180bddf](https://github.com/shivanigangadharan/e-mart/pull/5/commits/180bddf41e1c821776aaeabf6578dabc9fc2b09e)